src/UIViewController/UIViewController+Containment.m

```diff
@@ -18,7 +18,7 @@ - (void)ip_addChildViewController:(UIViewController *)controller toView:(UIView
     [controller willMoveToParentViewController:self];
```

Comment:
https://developer.apple.com/reference/uikit/uiviewcontroller/1621381-willmovetoparentviewcontroller

> When your custom container calls the addChildViewController: method, it automatically calls the willMoveToParentViewController: method of the view controller to be added as a child before adding it.

We can also drop this